packages/lit-html/src/lit-html.ts

```diff
@@ -1221,17 +1241,37 @@ class ChildPart {
     start: ChildNode | null = wrap(this._$startNode).nextSibling,
     from?: number
   ) {
-    this._$setChildPartConnected?.(false, true, from);
+    this._$notifyConnectionChanged?.(false, true, from);
     while (start && start !== this._$endNode) {
       const n = wrap(start!).nextSibling;
       (wrap(start!) as Element).remove();
       start = n;
     }
   }
+  /** @internal */
+  setConnected(isConnected: boolean) {
```

Comment:
I think we should still add some docs that say this should only be called on root parts.

Reply:
Done